src/CTA.WebForms2Blazor/FileConverters/ConfigFileConverter.cs

```diff
@@ -3,19 +3,57 @@
 using System.Text;
 using System.Threading.Tasks;
 using CTA.WebForms2Blazor.FileInformationModel;
+using CTA.Rules.Actions;
+using CTA.Rules.Models;
+using Newtonsoft.Json.Linq;
+using System.Configuration;
+using System.IO;
+using System.Reflection;
+using Newtonsoft.Json;
+using System.Linq;
 
 namespace CTA.WebForms2Blazor.FileConverters
 {
     public class ConfigFileConverter : FileConverter
     {
+        private const string webConfigFile = "web.config";
+        private string _projectPrefix;
+        private string _fullPath;
+        private string _relativeDirectory;
+
         public ConfigFileConverter(string relativePath) : base(relativePath)
         {
+            var workingDirectory = Environment.CurrentDirectory;
+            _projectPrefix = Directory.GetParent(workingDirectory).Parent.Parent.FullName;
+            _fullPath = Path.Combine(_projectPrefix, RelativePath);
+            _relativeDirectory = Path.GetDirectoryName(RelativePath);
```

Comment:
Environment.CurrentDirectory gives the working directory of the WebForms2Blazor project, not the WebFormsProject we're trying to migrate, if you need access to the full path maybe modifying the FileConverters to take both the relative path and the input base path as parameters is a good idea

Reply:
True, I'll change that